Gemfile

```diff
@@ -27,7 +27,7 @@ group :test do
   gem 'rspec-rails', '~> 3.7.0'
   gem 'capybara', '~> 2.13.0'
   gem 'poltergeist', '~> 1.14.0'
-  gem 'factory_girl_rails', '~> 4.8.0'
+  gem 'factory_bot_rails'
```

Comment:
I like to keep the gems locked to at least a minor version to avoid unintentional upgrades.

Since this gem is available starting at the 4.8.2 patch level, let's lock this to `gem 'factory_bot_rails', '4.8.2'`.

Reply:
Done. I have updated the PR.